exploration/outbreak_eda.ipynb

```diff
@@ -0,0 +1,273 @@
+    "                \"start_year\",\n",
+    "                \"ISO3\",\n",
+    "                \"admin1\",\n",
+    "                \"admin2\",\n",
+    "                \"location_period_id\",\n",
+    "            ]\n",
+    "        )[\"total_suspected_cases\"]\n",
+    "        .sum()\n",
+    "        .reset_index()\n",
+    "    )\n",
+    "    .merge(district_geometries, on=\"location_period_id\")\n",
+    "    .assign(location=lambda gdf: gdf.admin2 + \", \" + gdf.admin1 + \", \" + gdf.ISO3)\n",
+    "    .set_index(\"location_period_id\")\n",
+    ")\n",
+    "\n",
+    "yearly_cases_gdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will add a new column `outbreak` to represent presence of an outbreak in the district (outbreak = 1). This will be used to provide sums or counts of outbreaks and will later be used when we develop a machine learning model and need outbreak absence data points (outbreak = 0)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "yearly_cases_gdf[\"outbreak\"] = 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "yearly_cases_gdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Total number of georeferenced outbreaks within the pre-processed dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(yearly_cases_gdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exploratory data analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we will start exploring those counties that repeatedly see cholera outbreaks. We'll group the district data by both start year and country code. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "repeated_outbreaks = yearly_cases_gdf.groupby([\"start_year\", \"ISO3\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "repeat_outbreak_bar = px.bar(\n",
+    "    yearly_cases_gdf,\n",
+    "    x=\"ISO3\",\n",
+    "    y=\"outbreak\",\n",
+    "    color=\"start_year\",\n",
+    ")\n",
+    "\n",
+    "repeat_outbreak_bar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Mapping Cholera outbreaks from 2010-2019 at the district level\n",
+    "Below, we use `plotly` to map cholera outbreaks over time. This is two get a better sense, geographically, of where repeated outbreaks are occuring and to visualize any spatial autocorrelation between them (i.e., are areas that reapetedly experience outbreaks in closer proximity to each other?)"
```

Comment:
```suggestion
    "Below, we use `plotly` to map cholera outbreaks over time. This is to get a better sense, geographically, of where repeated outbreaks are occurring and to visualize any spatial autocorrelation between them (i.e., are areas that repeatedly experience outbreaks in closer proximity to each other?)"
```

Reply:
And this is why I wish VS code had spelling and grammar checking capabilities! ;) Thanks for catching these.